### DISCUSSION TITLE
Welcome to github-ospo Discussions!

### DISCUSSION THREAD

**Discussion by zkoppert** [github/github-ospo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do in the open source programs community.

**Comment by avelino** [comment]
I contribute to and maintain a few open source projects, but I confess that I have lost count of how many projects I have contributed to.

I've been in fantastic teams of open source projects, from core Go programming language to small libraries written in Racket (lisp dialect not very widespread outside the academic world).

I believe my GitHub account (@avelino) can explain my time investment in open source projects much better than me writing, I'm usually involved in projects that I use in my day-to-day work (currently written in Clojure, Go and Python).

It's a pleasure to be here and count on me to share knowledge and give opinions about doubts and problems that maintainers of open source projects are going through
